stix_shifter/stix_translation/src/utils/transformers.py

```diff
@@ -169,8 +169,53 @@ def transform(obj):
             print("Cannot convert input to Unix timestamp")
 
 
+class EpochToGuardium(ValueTransformer):
+    """A value transformer for Epoch to Guardium timestamp"""
+
+    @staticmethod
+    def transform(epoch):
+        return (datetime.fromtimestamp(int(epoch) / 1000, timezone.utc).strftime('%Y-%m-%d %H:%M:%S'))
+
+
+class GuardiumToTimestamp(ValueTransformer):
+    """A value transformer for converting Guardium timestamp to regular timestamp"""
+
+    @staticmethod
+    def transform(gdmTime):
+        rgx = r"(\d\d\d\d-\d\d-\d\d)\s(\d\d:\d\d:\d\d)"
+        mtch = (re.findall(rgx, gdmTime))[0]
+        return (mtch[0] + 'T' + mtch[1]) + '.000Z'
+
+
+class TimestampToGuardium(ValueTransformer):
+    """A value transformer for converting  regular timestamp to Guardium timestamp"""
+
+    @staticmethod
+    def transform(timestamp):
+        rgx = r"(\d\d\d\d-\d\d-\d\d).(\d\d:\d\d:\d\d)"
```

Comment:
A slightly more readable way of doing this would be:
`converted_string = re.sub("\.\d{3}Z$", "", timestamp)`
`converted_string = re.sub("T", " ", converted_string)`
`return converted_string`

Reply:
I'm referring to TimestampToGuardium since I think you are trying to remove the `T` and `.000Z` from the timestamp in the STIX pattern.